packages/controllers/src/permissions/README.md

```diff
@@ -181,6 +212,20 @@ permissionController.executeRestrictedMethod(origin, 'wallet_getSecret', {
 });
 ```
 
+### Getting Endowments
+
+```typescript
+// Getting endowments internally is the only option, since the host has to apply
+// them in some way external to the permission system.
```

Comment:
Nit: I found this comment a little confusing. I assume that "getting endowments internally" means getting them from within our codebase rather than via RPC method. It makes sense to me that endowments are internal, because... well... we only have a JSON-RPC API, and that doesn't include endowments as a concept (at least not directly).

But I don't know what "the host has to apply them in some way external to the permission system" means, or what its relevance is to whether endowments are exposed via RPC or not.